Role for forwarding server mail to SMTP server

jbzdak opened the pull request:
A role that sets up posix server that forwards all local mail to a designated email. 

Author's concerns: 
- If you have a shell account you can send emails from our server using the `mail` command. If bad people get accounts on our servers, this will be least of our concerns though.

antoviaque commented:
> If you have a shell account you can send emails from our server using the mail command. If bad people get accounts on our servers, this will be least of our concerns though.

Yes, if they have shell access they can send emails anyway unless we completely block outbound ports - and they can definitely do much more harm than than.

antoviaque commented on defaults/main.yml:
```diff
@@ -0,0 +1,15 @@
+---
+# defaults file for forward-mails-to-gmail
```
To GMail? probably a leftover

antoviaque commented:
:+1:

Commit:
Initial version of forward server mail role

Squashed commit of the following:

commit a198d6540ff883a7871a50a078eff60b5dd6e5a4
Author: Jacek Bzdak <jacek@opencraft.com>
Date:   Mon Apr 25 21:28:04 2016 +0200

    Fix typo

commit a6057ddc0d7e667049bb01c609ea1e5ff683ec14
Author: Jacek Bzdak <jacek@opencraft.com>
Date:   Mon Apr 25 12:17:07 2016 +0200

    Rewrite outgoing mail

commit 6d9b0c358376a7f5a95bf119bf02d5395add2e59
Author: Jacek Bzdak <jacek@opencraft.com>
Date:   Mon Apr 25 01:23:19 2016 +0200

    Renamed mailer-daemon

commit 1d52843306ac3ab57877d15921f7b68e0fc91ea6
Author: Jacek Bzdak <jacek@opencraft.com>
Date:   Sun Apr 24 22:21:10 2016 +0200

    Fix from address

commit 38d7a3408319837257a45bb168b35a053dd14d64
Author: Jacek Bzdak <jacek@opencraft.com>
Date:   Thu Apr 21 19:30:48 2016 +0200

    Fixed a default

commit c15247a884b8083c62d5def22542b83750871b29
Author: Jacek Bzdak <jacek@opencraft.com>
Date:   Thu Apr 21 18:24:24 2016 +0200

    PR review

commit bc8a0034fd29ec8b3d0c82c44a034749b90de8e1
Author: Jacek Bzdak <jacek@opencraft.com>
Date:   Mon Apr 18 14:24:33 2016 +0200

    Added meta dir

commit 840dd3de541aae5b41aa6115a28754851737b46e
Author: Jacek Bzdak <jacek@opencraft.com>
Date:   Mon Apr 18 12:55:43 2016 +0200

    Role for forwarding server mail to SMTP server

jbzdak commented:
Squashed and merging
